### DISCUSSION TITLE
Is it possible to use icons from the GTK theme?

### DISCUSSION THREAD

**Discussion by AlexNabokikh** [Alexays/Waybar#2880] [Q&A]
Hi everyone,

I was wondering if it's possible to use the icons for wifi, Bluetooth, network, battery and other indicators from my GTK theme. I am satisfied with the icons that the theme provides, and don't want to spend time searching for other icon sets.

Thank you!

**Comment by zaguragit** [comment]
This would be really great!!
I'm personally not a huge fan of the font-awesome icons, plus making your own icon theme (i made one) is much easier than ur own icon font, and its also just rly nice to have the icons be consistent with the ones used in every app respecting the icon theme

**Reply by zaguragit** [reply]
i managed to come up with a workaround on NixOS. I'm using image modules, with bash scripting in the `exec` field, and a convenience function (nixos specific) that gives me the path to the icon i want, for example:
```nix
"image#airplane-mode" = let
  get-icon = name: "${config.gtk.iconTheme.package}/share/icons/${config.gtk.iconTheme.name}/${name}.svg";
in {
  path = get-icon "scalable/status/airplane-mode-symbolic";
}
```
Sadly, this isnt rly a good solution cuz:
1. you have to explicitly declare a gtk icon theme with the package and name
2. the internal paths inside the icon themes vary, so if u change the theme ur icons are probably gonna dissapear and ull have to fix the paths
3. you have to write custom bash scripts if u want the icon to by dynamic
